### ISSUE TITLE
Encourage people to add topics/geolocations, in the profile (e.g. when saving)

### ISSUE THREAD

**Issue by karol-depka** [karol-depka/NetworkingMatcher#147]
E.g. via popup after saving.

Some people create profile but don't put topics / geo-locations / other-profiles.
We could have a popup or smth, to encourage Them...

Ideally, with some attractive graphics.

Adding @leannico as this seems like a nice way to get feet wet with Angular.

@PODAtron can you create some graphics to encourage people to add topics / other-profiles / geolocations to their profiles?

Thanks :).

**Comment by AlejandroMN** [comment]
If we analyze this situation in terms of **UX**, it is known that people dont like to fill in forms. 
Our profile creation process may be too discouraging considering the number of forms to fill in.
I recommend a full re-design of the **profile page**, where we move to a **stepper**.
"Complete your profile in 3 steps" . 
Adding new graphics/colors/fonts of course.
This approach will involve re-think the way topics are added.

**Comment by karol-depka** [comment]
Plz also link the issue where we track the topic popup dialog that Moises is doing (where you click pair-programming, hackathon, etc in relation to topic). Is there an issue for that already?

**Comment by AlejandroMN** [comment]
Yes, the second modal created by Moises should be considered here.

I encourage all the people in the team to do some proposal of the process. Mocks if posible
